src/main/resources/view/HelpWindow.fxml

```diff
@@ -41,92 +49,137 @@
-        </VBox>
-        <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER">
-          <Label fx:id="sortHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
-          <Label fx:id="sortHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
-          <padding>
-            <Insets bottom="5" left="10" right="10" top="5" />
-          </padding>
-        </VBox>
-        <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER">
-          <Label fx:id="viewHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
-          <Label fx:id="viewHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
-          <padding>
-            <Insets bottom="5" left="10" right="10" top="5" />
-          </padding>
-        </VBox>
-        <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER">
-          <Label fx:id="exitHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
-          <Label fx:id="exitHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
-          <padding>
-            <Insets bottom="5" left="10" right="10" top="5" />
-          </padding>
-        </VBox>
+
+        <!-- Commands for ClientHub -->
+        <ScrollPane style="-fx-background-color: white; -fx-border-color: transparent;" fitToWidth="true" fitToHeight="true">
+          <VBox style="-fx-background-color: white; -fx-border-color: transparent;" spacing="14">
+
+            <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER" spacing="5">
+              <Label fx:id="addHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
+              <HBox styleClass="commandWrapper">
+                <Label fx:id="addHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
+              </HBox>
+              <padding>
+                <Insets bottom="5" left="10" right="10" top="5" />
+              </padding>
+            </VBox>
+
+            <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER" spacing="5">
+              <Label fx:id="listHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
+              <HBox styleClass="commandWrapper">
+                <Label fx:id="listHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
+              </HBox>
+              <padding>
+                <Insets bottom="5" left="10" right="10" top="5" />
+              </padding>
+            </VBox>
+
+            <VBox alignment="CENTER_LEFT" styleClass="commandContainer" VBox.vgrow="NEVER" spacing="5">
+              <Label fx:id="editHelpHeader" alignment="CENTER_LEFT" styleClass="headerLabel" />
+              <HBox styleClass="commandWrapper">
+                <Label fx:id="editHelp" alignment="CENTER_LEFT" styleClass="commandLabel" />
+              </HBox>
+              <padding>
+                <Insets bottom="5" left="10" right="10" top="5" />
+              </padding>
+            </VBox>
```

Comment:
Good changes!

Reply:
I like these changes made to the UI! Big improvement from the original addressbook